client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningStyles.ts

```diff
@@ -5,6 +5,10 @@ const useStyles = makeStyles((theme: Theme) => ({
     form: {
         padding: '1vh 1.3vw 2.5vh 1.3vw',
     },
+    occupationSelect: {
+        width: '14vw',
+        height: '2vw'
+    },
```

Comment:
Why both vw and a grid wrapper? If the item is gridded give it fullWidth and if it's controlled by vw, don't grid it

Reply:
fixed